src/Form/WebdamConfig.php

```diff
@@ -51,6 +103,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#title' => $this->t('Password'),
       '#default_value' => $config->get('password'),
       '#description' => $this->t('The passwords of the Webdam account to use for API access. Note that this field will appear blank even if you have previously saved a value.'),
+      '#required' => TRUE,
```

Comment:
Are all of the other auth fields #required as well? If not, can you please make them required?